### DISCUSSION TITLE
Outline like in RStudio

### DISCUSSION THREAD

**Discussion by lixiang117423** [posit-dev/positron#4417] [Q&A]
Could we consider displaying the outline and the QMD file in the same interface, similar to RStudio?
![1](https://github.com/user-attachments/assets/9dd24ef8-5ec1-4136-bd80-09e01deb5ee5)

Thanks a lot!

**Comment by juliasilge** [comment]
You can check out how to set that up here: https://github.com/posit-dev/positron/discussions/4357#discussioncomment-10359437
